src/util/invariant.h

```diff
@@ -10,6 +10,8 @@ Author: Martin Brain, martin.brain@diffblue.com
 #define CPROVER_UTIL_INVARIANT_H
 
 #include <stdexcept>
+#include <type_traits>
+#include <iostream>
```

Comment:
Just to follow-up on that: the implementation of the function using `std::cerr` would thus have to go in the cpp file.